installers/helm/values.yaml

```diff
@@ -141,3 +141,27 @@ storage9_access_mode: "ReadWriteOnce"
 storage9_size: "1Gi"
 storage9_type: "dynamic"
 storage9_class: "rook-ceph-block"
+
```

Comment:
@cap1984 should we add it?

Reply:
@djtaylor for helm please check out https://github.com/percona/percona-helm-charts.
We consider removing crunchy helm from this very repo